x-pack/test/observability_ai_assistant_api_integration/tests/public_complete/public_complete.spec.ts

```diff
@@ -44,12 +44,19 @@ export default function ApiTest({ getService }: FtrProviderContext) {
     let proxy: LlmProxy;
     let connectorId: string;
 
-    async function getEvents(
-      params: {
-        actions?: Array<Pick<FunctionDefinition, 'name' | 'description' | 'parameters'>>;
-        instructions?: string[];
-      },
-      cb: (conversationSimulator: LlmResponseSimulator) => Promise<void>
+    interface RequestOptions {
+      actions?: Array<Pick<FunctionDefinition, 'name' | 'description' | 'parameters'>>;
+      instructions?: string[];
+      format?: 'openai';
```

Comment:
Should we have an enum/constant for valid formats?

Reply:
if we expand, sure. for now it's just in this place (contained in a single file and a test), so I figured it'd be overkill